src/Components/Common/Button/Button.js

```diff
@@ -1,7 +1,7 @@
 import classes from './Button.module.scss';
 
-export const Button = ({text, onClick}) => {
+export const Button = ({text, onClick, margin}) => {
```

Comment:
What will happen if you need to add paddings? =) Will you add new prop "padding"? =) 
It is better to create "className" prop, which will take class name and apply all necessary styles using just one prop =)

Reply:
Done!